arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -458,22 +471,87 @@ arangodb::aql::AqlValue aqlFnTokens(arangodb::aql::ExpressionContext* expression
   auto buffer = irs::memory::make_unique<arangodb::velocypack::Buffer<uint8_t>>();
 
   if (!buffer) {
```

Comment:
we don't need this check at all since `new` throws `std::bad_alloc` according to standard